FLT/DedekindDomain/FiniteAdeleRing/BaseChange.lean

```diff
@@ -376,6 +376,15 @@ noncomputable instance comap_pi_algebra (v : HeightOneSpectrum A) :
     Algebra (v.adicCompletion K) (Π (w : v.Extension B), w.1.adicCompletion L) :=
   (adicCompletionComapSemialgHom' A K L B v).toAlgebra
 
+attribute [local instance] comap_algebra comap_algebra_continuousSmul in
+instance comap_pi_continuousSmul (v : HeightOneSpectrum A) :
+    letI := comap_pi_algebra A K L B v |>.toSMul
+    ContinuousSMul (adicCompletion K v) (Π (w : v.Extension B), w.1.adicCompletion L) := by
```

Comment:
We're not very consistent on how to write `Π (w : v.Extension B), w.1.adicCompletion L`.  I've tried to match whatever the closest existing use had, but it's probably worth making them all match.